test/unit/steps/check-your-appeal/CheckYourAppeal.test.js

```diff
@@ -54,20 +54,20 @@ describe('CheckYourAppeal.js', () => {
 
   describe('sendToAPI()', () => {
     it('should log a message when successfully making an API call', () => {
+      appInsightsStub.trackTrace = sinon.stub().returns();
```

Comment:
This seems redundant as it's overridden two lines below

Reply:
updated thanks !